client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/style.scss

```diff
@@ -12,3 +12,43 @@ button.configure-contact__button {
 	border-color: var(--studio-gray-80);
 	padding: 4px 8px !important;
 }
+
+.configure-contact__form {
+	margin-block-start: 24px;
+
+	.notification-settings__footer-validation-error {
+		margin-block: 8px;
+	}
+}
+
+.configure-contact__help-text {
+	font-weight: 400;
+	font-size: 0.75rem;
+	line-height: 15px;
```

Comment:
I just learned this recently, but it is recommended to use unitless for line-height. The browser preserves the font-size/line-height ratio by making the line height unitless, even if the font size changes (Browser preferences).

Or If there are no hard design requirements for line heights, I suggest omitting this and letting the spacing work naturally.

Reply:
Thanks for letting me know. I think it makes sense to remove it - https://github.com/Automattic/wp-calypso/pull/78000/commits/6d43ded9f22ca5f25d88ad93a6ce94410556cec2